mdx/api/index.mdx

```diff
@@ -0,0 +1,776 @@
+| ------------       | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| sellToken          | The ERC20 token address or symbol of the token you want to send. "ETH" can be provided as a valid `sellToken`.                                                                                                      |
+| buyToken           | The ERC20 token address or symbol of the token you want to receive.                                                                                                                                                 |
+| sellAmount         | (Optional) The amount of `sellToken` (in `sellToken` units) you want to send. If `sellAmount` is provided, `buyAmount` will be ignored.                                                                                           |
+| buyAmount          | (Optional) The amount of `buyToken` (in `buyToken` units) you want. When both are provided, `sellAmount` takes precedence over this argument.                                                                                    |
+| slippagePercentage | (Optional, defaults to ".2") The maximum acceptable slippage in % of the `buyToken` amount if `sellAmount` is provided, the maximum acceptable slippage in % of the `sellAmount` amount if `buyAmount` is provided. |
+| gasPrice           | (Optional, defaults to ethgasstation "fast") The target gas price (in wei) for the swap transaction. If the price is too low to achieve the quote, an error will be returned.                                       |
+| takerAddress       | (Optional) The address which will fill the quote. When provided the gas will be estimated and returned. An eth_call will also be performed. If this fails a Revert Error will be returned in the response.          |
+
+
+### Response
+
+| Field       | Description                                                                                                                                                              |
+| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| price       | The price of 1 `buyToken` denoted in the `sellToken`. This price does not include the `slippage` provided in the request above.                                          |
+| to          | The address of the contract to send call `data` to.                                                                                                                      |
+| data        | The calldata required to be sent to the to contract address.                                                                                                             |
```

Comment:
I would love a link/more info to what `data` does -- from the lens of a frontend dev, I don't know what this is for